conf/config.yaml

```diff
@@ -111,8 +111,8 @@ nginx_config:                     # config for render the template to genarate n
     real_ip_from:                  # http://nginx.org/en/docs/http/ngx_http_realip_module.html#set_real_ip_from
       - 127.0.0.1
       - 'unix:'
-    lua_shared_dicts:
-      ipc_shared_dict: 100m         # custom shared cache
+    lua_shared_dicts:              # add custom shared cache to nginx.conf
+      ipc_shared_dict: 100m        # custom shared cache, format: `cache-key: cache-size`
```

Comment:
should be `# ipc_shared_dict: 100m`, which is only a demo.

Reply:
ok, thanks~